### PR TITLE
Add relevent links to other math functions [CNT-671]

### PR DESCRIPTION
**Pull request by aliphys**

### CODE REVIEW

**Comment by aliphys** [comment]
@per1234 could you review this?

**Comment by per1234** [comment]
It is somewhat a duplicate of https://github.com/arduino/reference-en/pull/816

The problem is that we used to have a very nice system that generated these links automatically. That system was broken and I have never been able to get a straight answer on whether it can simply be fixed rather than requiring us to manually replace and maintain a huge number of links just to get back to what we had for years.

**Comment by aliphys** [comment]
@per1234 If this PR is obsolete, could you close it? Thanks!

**Comment by per1234** [comment]
@aliphys since I still have not been able to get a straight answer after two years of trying, I think we must conclude that the automated links system will never be repaired and that it will be necessary to manually replace and maintain the links.

So this PR is not obsolete.

File: Language/Functions/Math/constrain.adoc [hunk 1]
```diff
@@ -88,5 +88,14 @@ int constrainedInput = constrain(input, minimumValue, maximumValue);
...
 === See also
 
+[role="definition"]
+* #DEFINITION# link:../../../variables/data-types/abs[abs]
+* #DEFINITION# link:../../../variables/data-types/map[map]
+* #DEFINITION# link:../../../variables/data-types/max[max]
+* #DEFINITION# link:../../../variables/data-types/min[min]
+* #DEFINITION# link:../../../variables/data-types/pow[pow]
+* #DEFINITION# link:../../../variables/data-types/sq[sq]
+* #DEFENITION# link:../../../variables/data-types/sqrt[sqrt]
```
Comment (per1234) [inline Language/Functions/Math/constrain.adoc:98]:
```suggestion
* #LANGUAGE# link:../abs[abs()]
* #LANGUAGE# link:../map[map()]
* #LANGUAGE# link:../max[max()]
* #LANGUAGE# link:../min[min()]
* #LANGUAGE# link:../pow[pow()]
* #LANGUAGE# link:../sq[sq()]
* #LANGUAGE# link:../sqrt[sqrt()]
```

- Correct links
- Use correct label: https://raw.githubusercontent.com/arduino/reference-en/master/AsciiDoc_sample/Reference_Terms/AsciiDoc_Template-Single_Entity.adoc
- Reproduce link text format from before the auto-link system was broken: https://web.archive.org/web/20190916201140/https://www.arduino.cc/reference/en/language/functions/math/abs/

Please make the equivalent fixes to all the other files in your PR.

**Review by per1234 — CHANGES_REQUESTED** [review]

File: Language/Functions/Math/constrain.adoc (hunk 1 above)
Comment (per1234) [reply Language/Functions/Math/constrain.adoc:98]: Hi @aliphys. I wanted to check back in with you. If you will update the pull request according to the instructions I provided above I will be happy to merge this important enhancement.

Comment (per1234) [reply Language/Functions/Math/constrain.adoc:98]: Hi @aliphys. I wanted to check back in with you. If you will update the pull request according to the instructions I provided above I will be happy to merge this important enhancement.

Comment (aliphys) [reply Language/Functions/Math/constrain.adoc:98]: Hello @per1234 , Thanks for the (multiple) follow ups. Much appreciated. I will check in with @karlsoderby when he comes back to see what his plans for the reference pages are.

Comment (per1234) [reply Language/Functions/Math/constrain.adoc:98]: Hi @aliphys. Any news?

Comment (aliphys) [reply Language/Functions/Math/constrain.adoc:98]: Just checked in with @karlsoderby , I'll make the changes you suggested by tomorrow.

File: Language/Functions/Math/abs.adoc [hunk 2]
```diff
@@ -72,5 +72,14 @@ a++;  // keep other math outside the function
 [float]
 === See also
 
+[role="definition"]
```
Comment (per1234) [inline Language/Functions/Math/abs.adoc:75]:
```suggestion
[role="language"]
```

According to the standardized reference sample:
https://github.com/arduino/reference-en/blob/master/AsciiDoc_sample/Reference_Terms/AsciiDoc_Template-Single_Entity.adoc?plain=1#L111

Please make the same change on the other pages as well.

**Review by per1234 — CHANGES_REQUESTED** [review]

File: Language/Functions/Math/constrain.adoc [hunk 3]
```diff
@@ -88,5 +88,14 @@ int constrainedInput = constrain(input, minimumValue, maximumValue);
 [float]
 === See also
 
+[role="definition"]
```
Comment (per1234) [inline Language/Functions/Math/constrain.adoc:91]:
```suggestion
[role="language"]
```

You missed this one.

**Review by per1234 — CHANGES_REQUESTED** [review]

**Review by per1234 — APPROVED** [review]
Thanks @aliphys!

**Pull request merged**